.github/workflows/facilities_build.yml

```diff
@@ -47,34 +46,30 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Run Container Setup
-        run: ../bash/docker_container_setup.sh
+        run: ./bash/docker_container_setup.sh
 
-      - name: Build Python Poetry deps and CLI
-        run: |
-          pip3 install poetry==1.5.1 &&\
-          python3 -m poetry config virtualenvs.create false --local && python3 -m poetry install --no-dev
+      - name: Install python deps
+        run: pip install -r ./python/requirements.txt
```

Comment:
per your note in the PR description, this is to use changes made to `requirements.txt` before the published docker image `nycplanning/build-geosupport:latest` incorporates those changes

yea I remember having this sort of chicken-egg problem before. definitely like your idea of building/using an image per branch. I think an approach at my last place was to make package changes in standalone PR's, but that had some issues